pipeline/clean/opuscleaner/configs/en-ja/default.filters.json

```diff
@@ -30,6 +30,69 @@
       },
       "language": null
     },
+    {
```

Comment:
I didn't notice before, but why don't we filter those for the opposite direction?

Reply:
My guess is we want the model to be able to translate regardless of the input punctuation. e.g. If we train only with fixed size punctuation, the model might struggle with website that might have ascii punctuation.